### DISCUSSION TITLE
Generating types in separate location.

### DISCUSSION THREAD

**Discussion by KrychuTM** [graphql-nexus/nexus-prisma#179] [Q&A] [answered]
Heya heya. Firstly I just wanna say thanks for making this plugin. Its been a huge help and very easy to use so keep up the good work. I'm pretty new to typescript and whilst reading the documentation for nexus-prisma I came across the Gentime Settings section and just wanted to know why would someone want to move the types file that is generated in the node_modules folder into their main project files? Is it just for ease of access or is there some benefit to having them there like better intellisense/code completion etc? Thanks 😊

**Comment by jasonkuhrt** [comment, marked as answer]
One functional reason is if the user has two prisma schemas in the project. In most cases the default works fine though.

**Reply by KrychuTM** [reply]
Oh okay thats good to know and thanks alot for the response!
